src/CRUD.js

```diff
@@ -1,335 +1,301 @@
+import { extend, intersect, storage } from './helpers.js';
+import StorageDriver from './StorageDriver.js';
+
 /**
  * Represents a database
  * @constructor
  * @param {Object} conf - the options to pass to the constructor
  */
-var Database = function ( conf ) {
-  this.conf = extend({
-    name: 'database',
-    indexedKeys: [],
-    uniqueKey: 'id',
-    driver: new Database.drivers.StorageDriver({
-      name: conf.name,
-      storage: exports.localStorage
-    })
-  }, conf || {});
-
-  this.initialize();
-};
-
-/**
- * Initialization method
- * @private
- */
-Database.prototype.initialize = function () {
-  this.data = this.load() || [];
-  this.id = 0;
-  if (this.data.length > 0) {
-    this.data = this.data.map(function(e) { return parseInt(e, 10); });
-    this.id = Math.max.apply(Math, this.data);
+class Database {
+  constructor( conf ) {
```

Comment:
`conf = {}`, thus removing `conf || {}` below.
Note it won't behave the same if someone passes a falsy value as `conf` so that's maybe not what you want.